demisto_sdk/commands/common/hook_validations/conf_json.py

```diff
@@ -75,3 +76,47 @@ def is_test_in_conf_json(self, file_id):
         if self.handle_error(error_message, error_code, file_path=CONF_PATH):
             return False
         return True
+
+    def is_valid_file_in_conf_json(self, current_file, file_type, file_path):
+        """Check if the file is valid in the conf.json"""
+        entity_id = _get_file_id(file_type.value, current_file)
+        if file_type == FileType.INTEGRATION:
+            return self.integration_has_unskipped_test_playbook(current_file, entity_id, file_path)
+        if file_type == FileType.SCRIPT:
```

Comment:
should also apply for playbooks.

we can also have it for core packs indicator types (there is a specific test for it)
@roysagi - WDYT?

Reply:
do we want all our playbooks to have active test playbooks from now on?

Reply:
I think we would do that as an enhancement but currently I think it is not something we want to pursue